src/Debug.mo

```diff
@@ -1,14 +1,32 @@
-/// Debugging aids
+/// Utility functions for debugging.
+///
+/// Import from the base library to use this module.
+/// ```motoko name=import
+/// import Debug "mo:base/Debug";
+/// ```
 
 import Prim "mo:⛔";
 module {
+  /// Prints `text` to output stream.
+  ///
+  /// NOTE: What this output stream is depends on your execution environment.
+  ///
+  /// ```motoko include=import
+  /// Debug.print "Hello New World!"
```

Comment:
As a new developer, I'd love for there to be an example in here using `debug_show`. That's a powerful tool for debugging at the moment.

Reply:
```suggestion
  /// Debug.print("numbers from 1 to 5: " # debug_show([1,2,3,4,5]));
```